### PR TITLE
Disabled authenticators are made un-droppable

### PR DESCRIPTION
**Pull request by vihanga-liyanage**
Screenshot from 2021-02-07 03-24-45

### CODE REVIEW

File: apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/authenticator-side-panel.tsx [hunk 1]
```diff
@@ -27,6 +27,7 @@ import { Authenticators } from "./authenticators";
 import { AppConstants, getOperationIcons, history } from "../../../../../core";
 import { GenericAuthenticatorInterface } from "../../../../../identity-providers";
 import { ApplicationManagementConstants } from "../../../../constants";
+import {log} from "util";
```
Comment (jeradrutnam) [inline apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/authenticator-side-panel.tsx:30]:
```suggestion
import { log } from "util";
```

File: apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/authenticators.tsx [hunk 2]
```diff
@@ -128,10 +128,43 @@ export const Authenticators: FunctionComponent<AuthenticatorsPropsInterface> = (
...
+                            "sections.signOnMethod.sections." +
+                            "authenticationFlow.sections.stepBased." +
+                            "secondFactorDisabled")
+                    }
+                </Text>
+            )
+        } else if (droppableId === ApplicationManagementConstants.EXTERNAL_AUTHENTICATORS_DROPPABLE_ID) {
+            return (
+                <Text>
+                    You need to configure this authenticator to use with your applications.
```
Comment (jeradrutnam) [inline apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/authenticators.tsx:162]: Shall we add this to a localize file?

File: apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/authenticator-side-panel.tsx [hunk 3]
```diff
@@ -27,6 +27,7 @@ import { Authenticators } from "./authenticators";
 import { AppConstants, getOperationIcons, history } from "../../../../../core";
 import { GenericAuthenticatorInterface } from "../../../../../identity-providers";
 import { ApplicationManagementConstants } from "../../../../constants";
+import { log } from "util";
```
Comment (brionmario) [inline apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/authenticator-side-panel.tsx:30]:
```suggestion
```

File: apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/authenticators.tsx (hunk 2 above)
Comment (vihanga-liyanage) [reply apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/authenticators.tsx]: Done

**Review by jeradrutnam — APPROVED** [review]

**Pull request merged**